migrations/Version20240405134821.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20240405134821 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql('ALTER TABLE "user" ADD oauth_azure_id VARCHAR(255) DEFAULT NULL');
+    }
+
+    public function down(Schema $schema): void
+    {
+        $this->addSql('CREATE SCHEMA public');
```

Comment:
I'm not sure why `doctrine:diff` is adding this line to the `down` method since the `public` schema already exists... any ideas?

Reply:
I created the migration by copy pasting the one where another oauth property was introduced, I asked myself why this line was there, but I left it as I couldn't find any plausible answer

Reply:
Very strange... please remove this line when you get a chance, I ran `bin/console doctrine:migrations:diff` locally between a fresh copy of main and your branch and it does not generate this... now I'm curious which migration you copied from, lol. Thank you!